.gitignore

```diff
@@ -10,3 +10,10 @@ pom.xml.asc
 /.nrepl-port
 .hgignore
 .hg/
+.cache
+.cpcache
+node_modules/
+package-lock.json
+package.json
```

Comment:
these two files should not be ignored

Reply:
They are being created automatically though, so I don't see a reason to have them in the repo

Reply:
hmm, I wouldn't ignore them and just commit them but do as you prefer